app/serializers/site_settings_serializer.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+class SiteSettingsSerializer < ApplicationSerializer
```

Comment:
Are we using this serializer?
We're returning a `Hash` for the `Admin::SiteSettings#index` and a value on the `SiteSettings#show`.

Reply:
It's not being used, I added it before and forgot to remove it. Removing it now.